meetings/Minutes_2016-01-14.md

```diff
@@ -1,11 +1,13 @@
-UofT Coders Executive Meeting
+## UofT Coders Executive Meeting
 Jan 13, 2016
 Attending: Madeleine, Luke, Lina
```

Comment:
These don't appear on separate lines when rendered. Maybe change to bullet points?